proto/lookout/sdk/service_data.proto

```diff
@@ -75,4 +76,5 @@ message FilesRequest {
     bool want_uast = 6 [(gogoproto.customname) = "WantUAST"];
     // WantLanguage set to true if UAST was requested
     bool want_language = 7;
+    repeated string include_languages = 9;
```

Comment:
why 9 instead of 8?

Reply:
copy-past from ChangesReq.
Fixed. Thanks!